t/api/02-assets.t

```diff
@@ -137,7 +137,6 @@ la;
 
 # test delete operation
 $t->delete_ok('/api/v1/assets/' . $listing->[0]->{id})->status_is(200);
-is($t->tx->res->json->{count}, 1, "one asset deleted");
```

Comment:
Why are you removing those checks?

Reply:
Because she changed the API and that actually makes sense if there weren't the concern about backward compatibility. (See my review comment.)